packages/fx-core/src/plugins/resource/apiconnector/plugin.ts

```diff
@@ -135,11 +145,38 @@ export class ApiConnectorImpl {
     await removeFileIfExist(sampleFilePath);
   }
 
+  private getAuthConfigFromInputs(inputs: Inputs): AuthConfig {
+    let config: AuthConfig;
+    if (inputs[Constants.questionKey.apiType] === AuthType.BASIC) {
+      config = {
+        AuthType: AuthType.BASIC,
+        UserName: inputs[Constants.questionKey.apiUserName],
+        Password: inputs[Constants.questionKey.apiPassword],
+      } as BasicAuthConfig;
+    } else if (inputs[Constants.questionKey.apiType] === AuthType.AAD) {
+      const AADConfig = {
+        AuthType: AuthType.AAD,
+      } as AADAuthConfig;
+      if (inputs[Constants.questionKey.apiAppType] === reuseAppOption.id) {
+        AADConfig.ReuseTeamsApp = true;
+      } else {
+        AADConfig.ReuseTeamsApp = false;
+        AADConfig.TenantId = inputs[Constants.questionKey.apiAppTenentId];
+        AADConfig.AppId = inputs[Constants.questionKey.apiAppId];
+      }
+      config = AADConfig;
+    } else {
+      throw new Error("todo");
```

Comment:
it's better throw ResultFactory error ?

Reply:
It is fixed. Thanks.
It will add more content next step.